### DISCUSSION TITLE
Previous button LR

### DISCUSSION THREAD

**Discussion by kjheemskerk** [rsjaffe/MIDI2LR#720] [General]
Hi! 

I am a frequent user of the previous button in Lightroom (the one where LR automatically copies the edit from previous photo and paste it on my current photo), but when trying to assign that task to midi2lr I can't seem to find it! 

Am I overlooking it or is it just not an option for the midi2lr software?

Thanks in advance!

**Comment by jamesmakes** [comment]
Hi! Did you find an answer to this? Currently having the same issue

**Reply by jakaerc** [reply]
in File - Plug-in Extras - General options - Other... you need to set a Keyboard shortcut of your choosing to Ctrl + Alt + V and in the MID2LR select that LR command.
